src/main/java/org/datadog/jenkins/plugins/datadog/DatadogUtilities.java

```diff
@@ -791,42 +791,22 @@ public static boolean isPipelineNode(FlowNode flowNode) {
         return flowNode instanceof FlowEndNode;
     }
 
-    /**
-     * Returns a normalize result for traces.
-     * @param result (success, failure, error, aborted, not_build, canceled, skipped, unknown)
-     * @return the normalized result for the traces based on the jenkins result
-     */
-    public static String getNormalizedResultForTraces(@Nonnull String result) {
-        switch (result.toLowerCase()){
-            case "failure":
-                return "error";
-            case "aborted":
-            case "not_built":
-                return "canceled";
-            default:
-                return result.toLowerCase();
-        }
-    }
-
     /**
      * Returns a normalized result for traces.
-     * NOTE: This is very similar to the above "getNormalizedResultForTraces", but with the difference
-     * that this will not return "unstable", which isn't a valid status on the Webhooks API.
      * @param result (success, failure, error, aborted, not_build, canceled, skipped, unknown)
      * @return the normalized result for the traces based on the jenkins result
      */
-    public static String statusFromResult(@Nonnull String result) {
-        switch (result.toLowerCase()){
+    public static String statusFromResult(String result) {
+        String resultLowercase = result == null ? "" : result.toLowerCase();
```

Comment:
This means the result might be null now? Are we supporting "empty" status? Or there is a fallback status

Reply:
I added this check because result is actually nullable, the Jenkins docs say that result might be null for unfinished jobs. I don't think this is ever the case for us, but I thought on adding a fallback. I agree that maybe it should be something other than empty string. What about `error`?

Reply:
```suggestion
        String resultLowercase = result == null ? "error" : result.toLowerCase();
```